ios/Plugin/UIImage.swift

```diff
@@ -0,0 +1,56 @@
+extension UIImage {
```

Comment:
This was just moved to a separate file. Was previously located in `CameraController.swift`